discord/user.py

```diff
@@ -61,6 +61,9 @@ def hypesquad(self):
     def partner(self):
         return self._has_flag(UserFlags.partner)
 
+    @property
+    def hypesquad_houses(self):
+        return [house for house, flag in zip(HypesquadHouse, (64, 128, 256)) if (self.flags & flag) == flag]
```

Comment:
don't hardcode this

Reply:
user flags are already hard coded in the enums

Reply:
i think maybe they should be their own enum, or maybe the powers of two could replace the values in HypesquadHouse

Reply:
Seconding, this needs to be done better. These are, after all, user flags that have their own enum.